policyengine_us/tests/policy/baseline/gov/states/ct/tax/income/credits/property_tax/ct_property_tax_credit.yaml

```
@@ -0,0 +1,101 @@
+- name: SINGLE household with $49,000  agi and  $100 property tax and eligible for property tax credit
```

Please format others similarly
```suggestion
- name: Single eligible filer with $49,000 AGI and $100 property tax
```